### DISCUSSION TITLE
Tolong bantu bang

### DISCUSSION THREAD

**Discussion by Zza225** [deaafrizal/forum#53] [The Lounge]
Mau nanya dong bang cara pake extansions di vscode gimana ya? saya sudah install tapi gabisa dipake 
![WhatsApp Image 2024-01-08 at 16 27 18_e6df1209](https://github.com/deaaprizal/forum/assets/155959251/2f88e0c3-028b-4388-b755-69cbe9a6f43e)

**Comment by mohnaufal2023** [comment]
control+shift+p baru nanti cari nama extentions nya, kemudian pilih mau pake yang mana dari extentetions "cuy University" tersebut

**Reply by Zza225** [reply]
udah di pake extentionsny tapi eror kenapa yaa?

**Reply by mohnaufal2023** [reply]
coba instal lagi aja bg , sama cek vidio tutor nya biar lebih jelas

**Reply by Zza225** [reply]
vidionya yang mana ya bang 🙏

**Reply by mohnaufal2023** [reply]
https://youtu.be/kh0Kv_XzM-g?si=N0hZGVA0CCmyPArM

**Reply by mohnaufal2023** [reply]
bg aku folow github abg gpp ya hehe

**Reply by Zza225** [reply]
Makasih banyak ya bang, kebantu banget ni

**Comment by Mikael2909** [comment]
itu sudah didownload extension cuy universe template nya
untuk menggunakannya tinggal tekan ctrl+shift+p lalu ketik cuy universe lalu klik add cuy universe habis itu pilih mau starter package nya mau baby pack, starter pack dll

**Reply by Zza225** [reply]
kalo eror kenapa ya?

**Reply by Mikael2909** [reply]
jaringan wifi aman ga? itu bisa karena pengaruh jaringan

**Comment by Zza225** [comment]
eror bang gabisa ke apply kenapa yaa?

**Reply by mohnaufal2023** [reply]
di start dulu bg exten ny yg baru di install baru pilih kategori yg mau di gunakan

**Comment by rezaprtma391** [comment]
mau pake extension yang mana? codenap ? pencet aja ctrl + shift + p terus block kode yang mau di ss nah kan muncul tuh view sebelah kanan tinggal pencet gambar foto
